### PR TITLE
add max prop instructions for <keep-alive>

### PR DESCRIPTION
**Pull request by wuyuweixin**

### CODE REVIEW

**Comment by phanan** [comment]
While the idea is good, I'm afraid the quality of the PR isn't optimal. Take a look at the Netlify build and you'll see what I mean :)

**Comment by sdras** [comment]
Looks like the trouble is not closing the html triple backticks

File: src/v2/api/index.md [hunk 1]
```diff
@@ -2516,6 +2516,16 @@ Used to denote a `<template>` element as a scoped slot, which is replaced by [`s
 
 - **See also:** [Dynamic Components - keep-alive](../guide/components.html#keep-alive)
 
+>New in 2.5.0+
+
+`max` maximum number of components allowed for caching (FIFO)
```
Comment (Justineo) [inline src/v2/api/index.md]: We should add a corresponding description into the `props` section of `keep-alive`.

```md
  - `max` - number. The maximum number ... . (new in 2.5.0+)
```

File: src/v2/api/index.md [hunk 2]
```diff
@@ -2516,6 +2516,16 @@ Used to denote a `<template>` element as a scoped slot, which is replaced by [`s
 
 - **See also:** [Dynamic Components - keep-alive](../guide/components.html#keep-alive)
 
+>New in 2.5.0+
```
Comment (Justineo) [inline src/v2/api/index.md]: Missing a space after `>`.

Comment (Justineo) [inline src/v2/api/index.md]: We can add a section similar to the previous “`include` and `exclude`”.

File: src/v2/api/index.md [hunk 3]
```diff
@@ -2516,6 +2516,16 @@ Used to denote a `<template>` element as a scoped slot, which is replaced by [`s
...
+
+`max` maximum number of components allowed for caching (FIFO)
+   ``` html
```
Comment (Justineo) [inline src/v2/api/index.md]: Indent with 2 spaces instead of 3 here.

File: src/v2/api/index.md [hunk 4]
```diff
@@ -2516,6 +2516,16 @@ Used to denote a `<template>` element as a scoped slot, which is replaced by [`s
...
+  <!-- usage -->
+  <keep-alive max="10">
+    <component></component>
+  </keep-alive>
+   ```
```
Comment (Justineo) [inline src/v2/api/index.md]: As above

Comment (wuyuweixin) [reply src/v2/api/index.md]: OK

**Comment by chrisvfritz** [comment]
Thanks for this and sorry for the delay in reviewing. I wanted to add more detail and different examples, so I just added this to the API doc in a separate commit. 🙂

**Pull request closed without merging**